docs/dictionary/command/import-widget.lcdoc

```diff
@@ -10,7 +10,7 @@ Introduced: 8.0
 
 OS: mac,windows,linux,ios,android
 
-Platforms: desktop,web,modile
+Platforms: desktop,web,mobile
```

Comment:
But how will I deply the app to my modile?... you very nearly deleted `web,` ;-)